tests/test_data_manager.py

```diff
@@ -153,26 +132,32 @@ def test_no_absolute_path_in_get_testdata_files(self):
             get_testdata_files("/foo/*.dcm")
 
 
-@pytest.mark.skipif(
-    not EXT_PYDICOM or not IS_WRITEABLE, reason="pydicom-data not installed or RO"
-)
-class TestExternalDataSource:
-    """Tests for the external data sources."""
+@pytest.fixture
+def data_fs(fs):
+    # Note: when using pyfakefs you must use open() not Path.open()
+    #   and os.remove() not Path.unlink()
```

Comment:
Hm - is this a bug in pyfakefs? I need to check this...

Reply:
Seems to be? The file kept being deleted from the actual FS with `p.open()`.

Reply:
Hmm, it seems like `pytest-xdist` doesn't play particularly nicely with `fs.add_real_directory()` due to the contents being added multiple times with each worker. 

Which might be even more annoying since I was hoping to create a separate FS for each worker to prevent them affecting each other.

Reply:
Which test did this happen? `test_get_testdata_file_external_hash_mismatch`?